src/petter_hetland_ex/ex04/walker.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Petter Hetland'
+__email__ = 'pehe@nmbu.no'
+
+import random as r
+
+
+class Walker:
+    def __init__(self, x_0, h):
+        self.x = x_0
+        self.h = h
+        self.num_steps = 0
+
+    def move(self):
+        if r.randint(0, 1) > 0.5:
+            self.x += 1
+        else:
+            self.x -= 1
+
+        self.num_steps += 1
+        return self.x, self.num_steps
+
+    def get_position(self):
+        return self.x
+
+    def is_at_home(self):
+        if self.x == self.h:
```

Comment:
you can replace this with a single line: 
return self.x == self.h

Reply:
Thanks Ahmed, this is really helpful. Changes made according to comments.